base_url trailing slash fix

eugene-katsevman opened the pull request:
I was not able to work with apiary's mock server until I added this little fix

eugene-katsevman committed:
trailing slash in base_url fixed

eugene-katsevman committed:
actual trailing slash in base_url fix

bpmason1 commented:
@eugene-katsevman sorry for the delay in merging this ... I was away on vacation for most of January.

eugene-katsevman commented:
That's ok. I'll send you more requests soon, I've added some functionality (adding/editing budgets and campaigns) and fixed some other strange things.

bpmason1 commented:
Awesome!!!!  I appreciate the help.
